.circleci/setup-system

```diff
@@ -13,11 +13,17 @@ sudo apt-get update
 #
 sudo apt-get install -y \
   default-jdk \
+  nodejs \
+  npm \
   python3 \
   python3-pip \
   python-is-python3 \
   scala
 
+# For atdts
+sudo npm install -g typescript
```

Comment:
it's a bit suspicious to have it install globally. I think that it should be a dev depedency in a package.json and then have a `script` field in this same package.json that would be like

```
"scripts": {
  "tsc": "tsc"
}
```

Reply:
done.